week-1-practise/level-2/2-format-currency.js

```diff
@@ -6,7 +6,10 @@
   Write a function that transforms numbers into the format £0.00
 */
 
-function formatCurrency() {}
+function formatCurrency(currency) {
+  var result = "£" + currency.toFixed(2);
+  return result
```

Comment:
Great way to use the .toFixed() method, I see you are enjoying coding, well done